client/gutenberg/extensions/spoiler/view.js

```diff
@@ -4,3 +4,14 @@
  * Internal dependencies
  */
 import './style.scss';
+
+function toggleSpoiler( event ) {
+	if ( event.target.matches( '.wp-block-a8c-spoiler' ) ) {
+		event.target.classList.toggle( 'is-visible' );
+	}
+}
+
+document.addEventListener( 'DOMContentLoaded', () => {
+	//add click handler on body to check click events for any spoiler tags
+	document.body.addEventListener( 'click', toggleSpoiler );
```

Comment:
going whole hog on this one eh?

Reply:
It's for cases where post content is inserted in after page load. Say we edit with p2 inline, `x` out and render the post again.

No strong feelings here though if we'd like to instead bind per element on DOM ready and have missing functionality in some cases.